### PR TITLE
docs: enrich the docs with more specific usage info

### PR DESCRIPTION
**Pull request by aviadhaham**
@orellazri please check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,13 +63,20 @@ You need to run the `renderkit` command with the following arguments as either c
...
 | `output`               | Output directory to write to                                                   | string |
-| `datasource`           | Datasource to use for rendering (scheme://path)                                | list   |
+| `datasource`           | Datasource to use for rendering (scheme://path) __**__                         | list   |
 | `data`                 | Data to use for rendering. Can be used to provide data directly                | list   |
 | `engine`               | Templating engine to use for rendering (Go Templates by default)               | string |
 | `allow-duplicate-keys` | Allow duplicate keys in datasources. If set, the last value found will be used | bool   |
 
+### **Notes on `datasource`
+
+- For now, only the `env` scheme is supported for datasources.
```
Comment (orellazri) [inline README.md:75]: Not very clear, should say that for plain files a scheme is not needed

File: README.md [hunk 2]
```diff
@@ -63,13 +63,20 @@ You need to run the `renderkit` command with the following arguments as either c
 | `input`                | Template string to render                                                      | string |
 | `input-file`           | Template input file to render                                                  | string |
 | `input-dir`            | Template input directory to render                                             | string |
-| `exclude`              | Glob patterns for files to exclude from rendering                              | list   |
+| `exclude`              | Exclude files/directories using path-based glob patterns                       | list   |
```
Comment (orellazri) [inline README.md:66]: Change flag help text in CLI as well

**Review by orellazri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aviadhaham) [reply README.md:75]: right!

**Comment by aviadhaham** [comment]
please check again after fixes @orellazri

**Pull request merged**